LL-6516 - Implement reactive route logic based on path + History integration

imqdee opened the pull request:
<!-- Description of what the PR does go here... screenshot might be good if appropriate -->
Implement reactive route logic based on path.

~Note that the UI for the history screens is currently broken. Have to discuss with the design to decide if we keep the exact same UI from v1 or if some changes are planned. However, logic works.~
**Edit**: UI fixed, at least for loading and placeholder screens. Have to take a look to the screen that list swap history in the future (doesn't have enough founds to make a swap at the moment when I'm writing)

### Type

Chore
<!-- e.g. Bug Fix, Feature, Code Quality Improvement, UI Polish... -->

### Context

[LL-6516](https://ledgerhq.atlassian.net/browse/LL-6516)

~~Awaiting #11~~

<!-- e.g. GitHub issue #45 / contextual discussion -->

### Parts of the app affected / Test plan

Swap screens.

### Attachment 📹 📷 

<details>
<summary>Light/Dark mode general screen recording</summary>

https://user-images.githubusercontent.com/33158502/127011270-c70107d2-6e46-4593-8b36-aa2559803694.mov


</details>

<details>
<summary>Light/Dark mode - Loading history screen recording</summary>


https://user-images.githubusercontent.com/33158502/127012524-55420932-b602-496e-9582-c31282a806c9.mov




</details>


<!-- Which part of the app is affected? What to do to test it, any special thing to do? -->

imqdee committed:
chore(swap): create navbar component

imqdee committed:
chore(swap):integrate routing logic based on path

juan-cortes commented on src/renderer/screens/exchange/Swap2/History/index.js:
```diff
@@ -0,0 +1,12 @@
+ ** For the moment, UI is broken and need to be fixed. I wait to discuss
+ ** with the design team before fixing it.
+ */
+const History = () => <HistoryV1 />;
```
Since we are forking away from v1, don't import anything from v1 but rather copy it and we will modify it inside the v2. As far as the UI being broken, it should at least take the full width/height of the component and render the placeholder, not this small centered version

imqdee replied:
Got it.

juan-cortes reviewed (changes requested):
Once you address the comments on #11 we can rebase here and cleanup before a merge

imqdee committed:
design(swap) includes v1 of the history screen

Copy the files from the v1 history directory to the v2 directory and makes
minor ui adjustments to adapt to the new layout.
Didn't touch anything about the logic.

imqdee committed:
i18n(swap): remove unused bullets translation

As asked [here](https://github.com/juan-cortes/ledger-live-desktop/pull/11/files/893e65e456abb967c8e419a90607908061295f10#r676534339)

imqdee commented:
Pushed changes related to the history page.
Loading and placeholder are integrated like a charm. Do you know if some UI changes are planned for these screens?
I'm going to send a new message to #live-faucet Slack's channel because I don't have enough tokens to use swap functionality so I didn't test the operation list view.

In addition to that, I updated my initial message by including a new screen recording.

juan-cortes commented on src/renderer/screens/exchange/Swap2/History/HistoryPlaceholder.js:
```diff
@@ -0,0 +1,32 @@
+  return (
+    <Wrapper>
+      <Text ff="Inter|SemiBold" fontSize={16} color="palette.text.shade100">
+        {t("swap.history.empty.title")}
```
swap2?

juan-cortes commented on src/renderer/screens/exchange/Swap2/History/History.js:
```diff
@@ -0,0 +1,181 @@
+          <Box>
+            <Head px={20} py={16}>
+              <Alert type="primary">
+                <Trans i18nKey="swap.history.disclaimer" />
```
swap2 all the history wording should be duplicated so we can just drop the old one when we finally switch

imqdee replied:
Done! @juan-cortes

juan-cortes reviewed (changes requested):
just the wording indexes and we're good

imqdee committed:
fix(swap/i18n): move from v1 translation to v2
